src/Microsoft.DotNet.Interactive.Formatting/PocketViewTags.cs

```diff
@@ -88,10 +88,11 @@ public static class PocketViewTags
         public static dynamic ul => _.ul;
         public static dynamic video => _.video;
 
-        // This can be used to indicates points
-        // where we are embedding an arbitrary value into the HTML content
-        // and we are explicitly indicating that the value can be either IHtmlContent
-        // or will otherwise be formatted as plaintext.
-        public static object arbitrary(object value) => value;
+        /// <summary>Create an object suitable for delayed expansion to HTML</summary>
+        public static object embed(object obj, FormatContext context)
+        {
```

Comment:
This method probably belongs in a different class and with a Pascal-cased name, as it seems to sit outside the DSL.

Reply:
I think it's part of the DSL as things stand, because it's used in DSL expressions and creates the special EmbeddedContent object later processed by the DSL writer.

But yes, this thing is somewhat needlessly delayed in its execution and could be taken outside the DSL.

Reply:
From an education point of view we would teach it as part of the DSL - "to embed an arbitrary object with a context use `embed`"

That reminds me, I'll need to update the C# Formatter.Register samples in the notebooks to propagate the context